pkg/cluster/config/v1alpha2/zz_generated.conversion.go

```diff
@@ -83,5 +83,6 @@ func autoConvert_config_Node_To_v1alpha2_Node(in *config.Node, out *Node, s conv
 	out.Role = NodeRole(in.Role)
 	out.Image = in.Image
 	out.ExtraMounts = *(*[]cri.Mount)(unsafe.Pointer(&in.ExtraMounts))
+	// WARNING: in.ExtraPortMappings requires manual conversion: does not exist in peer-type
```

Comment:
I think we can just drop v1alph2 for 0.4 to nudge things forward, I'm starting to think about v1alpha4 :-)

Reply:
(this PR is fine, but this is a good reminder to drop that version)